libs/utils/trace.py

```diff
@@ -629,6 +631,13 @@ def _sanitize_SchedEnergyDiff(self):
             "Maximum estimated system energy: {0:d}".format(power_max))
 
         df = self._dfg_trace_event('sched_energy_diff')
+
+        translations = {'nrg_d': 'nrg_diff', 'utl_d': 'usage_delta'}
+        if any(column in df for column in translations):
+            for new_name, old_name in translations.items():
+                df[old_name] = df[new_name]
+                del df[new_name]
+
```

Comment:
Here you are copying data... talking about optimising DataFrame usage, perhaps something like that should work:  

```python
translations = {'nrg_diff' : 'nrg_d', 'usage_delta' : 'utl_d'}
if any(column in df for column in translations):
    df.rename(columns=translations, inplace=True)
```